pkgs/by-name/de/decent-sampler/package.nix

```diff
@@ -10,22 +12,40 @@
 
 let
   pname = "decent-sampler";
-  version = "1.9.4";
+  version = "1.10.0";
 
-  decent-sampler = stdenv.mkDerivation {
+  decent-sampler = stdenv.mkDerivation rec {
```

Comment:
don't think this change is necessary, `icon` could be moved out into the `let` block.